css/styles.css

```diff
@@ -0,0 +1,16 @@
+/** @format */
+#header {
+  display: flex;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  left: 0;
+}
+
+@media (max-width: 1200px) {
+  #header {
```

Comment:
I think it would be better to use a media query for the mobile version as well. Because having a mobile-responsive website boosts the SEO value of a website as Google prefers indexing and ranking websites that are mobile-friendly. Incorporating a responsive layout goes a long way in assisting websites in ranking higher on Google searches.